frontend/cypress/integration/ChooseInterestsPageController_spec.js

```diff
@@ -0,0 +1,22 @@
+describe('ChooseInterestsPageController', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    cy.get('button').contains('Create an Account').click()
+
+    const time = Date.now()
+    cy.get('input').eq(0).type(`${time}`)
+    cy.get('input').eq(1).type(`${time}@testing.com`)
+    cy.get('input').eq(2).type('password')
+    cy.get('button').contains('Continue').click()
+  })
+
+  it('shows the choose interests page after signing up', () => {
+    cy.url().should('eq', 'http://localhost:3000/signUp')
+    cy.get('h5').contains('What are you interested in?').should('be.visible')
+  })
+
+  it('should redirect to dashboard once the continue to dashboard button is clicked', () => {
+    cy.get('#continue-to-dashboard').click()
+    cy.url().should('eq', 'http://localhost:3000/')
+  })
+})
```

Comment:
Let's move this file to `frontend/cypress/integration/ChooseInterestsPage/ChooseInterestsPage.test.js`

Reply:
also change the beforeEach to use the cy.login() method like in other tests.

Reply:
Would I need to change my `beforeEach` because I am trying to register a user manually by going through the UI?